lib/OpenQA/Worker/Job.pm

```diff
@@ -1222,19 +1232,19 @@ sub _log_snippet ($self, $file, $offset_name) {
     return \%ret;
 }
 
-sub _optimize_image ($image, $job_settings) {
+sub _optimize_image ($image, $job_settings, $optipng_bin = 'optipng', $pngquant_bin = 'pngquant') {
+    my $optimize_flag = $job_settings->{OPTIMIZE_IMAGES};
```

Comment:
This flag is never documented. Can you put that into the error message as in "optipng failed. You can set OPTIMIZE_IMAGES=0 to disable"